iica-catie.csl

```diff
@@ -414,21 +438,76 @@
           <text macro="URL" prefix="Disponible en "/>
           <text macro="container-title" prefix=" (" suffix="). "/>
         </else-if>
+        <else-if type="paper-conference" match="any">
+          <text macro="author-short" suffix=". "/>
+          <text macro="year-date" suffix=". "/>
+          <text macro="title" suffix=" "/>
+          <choose>
+            <if match="any" variable="URL">
+              <text value="(en línea)" suffix=". "/>
+            </if>
+          </choose>
+          <choose>
+            <if match="any" variable="editor">
+              <names variable="editor" suffix=" (ed.). "/>
+              <text variable="container-title" suffix=". "/>
+            </if>
+            <else>
+              <text variable="container-title" suffix=". "/>
+            </else>
+          </choose>
+          <text macro="publisher-place" suffix=", "/>
+          <text macro="publisher" suffix=". "/>
+          <text macro="number-of-pages"/>
+          <choose>
+            <if match="any" variable="URL">
+              <text macro="access" prefix="Consultado "/>
+              <text macro="URL" prefix="Disponible en "/>
+            </if>
+          </choose>
+        </else-if>
+        <else-if type="patent" match="any">
+          <text macro="author-short" suffix=" (inventor); "/>
+          <text macro="year-date" suffix=". "/>
+          <text macro="title" suffix=". "/>
+          <choose>
+            <if match="any" variable="URL">
+              <text value="(en línea)" suffix=". "/>
+            </if>
+          </choose>
+          <text macro="publisher-place" suffix=", "/>
+          <text macro="publisher" suffix=". "/>
+          <text variable="number" suffix=". "/>
+          <date variable="issued">
+            <date-part name="day" suffix=" "/>
+            <date-part name="month" form="short"/>
+          </date>
+          <text variable="page" display="block" suffix=" p. "/>
```

Comment:
`display="block"` should be deleted here as well.

Reply:
ok